pages/recipeEdit/index.js

```diff
@@ -470,6 +481,12 @@ class EditRecipePage extends React.Component {
       >
         <div className="cmpsr-edit-actions pull-right">
           <ul className="list-inline">
+            <li className="text-muted">
+              3 changes
+            </li>
+            <li>
+              <a onClick={(e) => this.handleShowModal(e, 'modalPendingChanges')}>View and Comment</a>
```

Comment:
The UI needs to be keyboard accessible for general accessibility purposes. For an <a> tag to be keyboard accessible, it needs `href="#"`. Alternativley, `tabindex="0"` will also work, but for <a> I've been using the href attribute.